api/src/pcapi/routes/backoffice/templates/offer/details/details.html

```diff
@@ -0,0 +1,86 @@
+{% import "components/links.html" as links %}
+{% from "components/presentation/details/tabs.html" import build_details_tab %}
+{% from "components/presentation/details/tabs.html" import build_details_tabs_wrapper %}
+{% from "components/presentation/details/tabs.html" import build_details_tab_content %}
+{% from "components/presentation/details/tabs.html" import build_details_tabs_content_wrapper %}
+{% from "components/turbo/spinner.html" import build_loading_spinner with context %}
+{% extends "layouts/standard.html" %}
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% block main_content %}
+  <div class="row row-cols-1 g-4 py-3">
+    <div class="col">
+      <div class="card shadow">
+        <div class="card-body">
+          <div class="row justify-content-start align-items-center">
+            <div class="col d-flex align-items-center justify-content-start">
+              <h2 class="card-title mb-3 text-primary">{{ links.build_offer_name_to_pc_pro_link(offer) }}</h2>
+            </div>
+            <div class="col-2"></div>
+          </div>
+          <p class="card-subtitle text-muted mb-3 h5">Offer ID : {{ offer.id }}</p>
+          <div class="row pt-3">
+            <div class="col-4">
+              <div class="fs-6">
+                <p class="mb-1">
+                  <span class="fw-bold">Catégorie :</span> {{ offer.category.pro_label }}
+                </p>
+                <p class="mb-1">
+                  <span class="fw-bold">Sous-catégorie :</span> {{ offer.subcategory.pro_label }}
+                </p>
+                <p class="mb-1">
+                  <span class="fw-bold">Date de création :</span> {{ offer.dateCreated | format_date }}
+                </p>
+                <p class="mb-1">
+                  <span class="fw-bold">Date de l'événement :</span>
+                  {% if offer.collectiveStock %}{{ offer.collectiveStock.beginningDatetime | format_date }}{% endif %}
```

Comment:
On est dans la page de détails d'une offre individuelle, dans laquelle on ne peut pas avoir de `collectiveStock`.
Voulais-tu plutôt cela ?
```suggestion
                  {% if offer.stock %}{{ offer.stock.beginningDatetime | format_date }}{% endif %}
```
Du coup, ça veut dire qu'il manque un test auto avec une offre d'événement :)